docs/user_interface.rst

```diff
@@ -25,284 +25,416 @@ If you do processing or acquisition which produces new data, its associated disp
 
-.. see https://github.com/nion-software/nionswift/issues/758
+* :guilabel:`Channels` - Change the range of x values shown on the line plot. These values are set to automatically calculate by default, but changing them allows you to zoom into a specific section on the line plot. Changing the channel values is similar to zooming into an interval on the line plot with an interval graphic.
 
-The fill color, stroke color, and stroke width control the look of the layer in the line plot. You can click on the color wells to bring up a color picker. You can enter colors as "#F00", "#00FF00", "blue". Clearing the field will make the color tranparent (not displayed). The default stroke width is 1.
+* :guilabel:`Log Scale Y` - Set the y axis to scale logarithmically.
 
-If your data is complex, you can also choose how to convert the data from complex to a scalar value for display. The options are Log Absolute, Absolute, Real, and Imaginary. The default is Log Absolute.
+* :guilabel:`Legend Position` - Choose the position of the legend for a layered line plot. Line plots with no layers will not show a legend. 
 
 .. _Data Info Inspector Section:
 
 Data Info
-+++++++++
-.. for each data item
-.. date, data description, shape, data type
+---------
+Data Info displays the date and time a selected line plot was created. It will also display the dimensions of the line plot and the bit count. 
 
-For each data item displayed with the display item, the :guilabel:`Data Info` inspector subsection shows you the creation date, description of the dimensional groupings, shape of the data, and the data type.
+.. image:: graphics/inspector_data_info_subsection.png
+    :width: 321
+    :alt: Inspector Data Info Subsection
 
-For image displays, there will only be one data item. For line plots will be one or more data items.
+For line plots with multiple layers, each data item in the stack will have its own Data Info section. For more information, see :ref:`layers inspector section`.
 
 .. _Calibrations Inspector Section:
 
 Calibrations
-++++++++++++
-.. for each data item
-.. offset, scale, units for each dimension
-.. displayed units
+------------
+Calibrations allow you to make specific changes to the scale and position of a selected item. Images and line plot displays have different features in the calibrations subsection.
+
+Image Calibrations
+++++++++++++++++++
+When an image is selected, the calibrations subsection of the inspector panel will show variables specific to an image.
+
+.. image:: graphics/inspector_calibrations_subsection_image.png
+    :width: 321
+    :alt: Inspector Calibrations Subsection with Image Selected
+
+With an image selected, use the calibrations subsection to
+
+* Change the offset, scale, and units on the y and x axes. The default units for images is nanometers (nm).
```

Comment:
The formula we use is x' = x * scale + offset. Add this explanation since there is another way it can be done: x' = (x + offset) * scale (this is NOT the way we do it).